src/session.rs

```diff
@@ -237,6 +287,62 @@ impl SessionWrapper {
             paging_state: paging_state.into(),
         })
     }
+
+    #[napi]
+    pub async fn execute_multiple(
+        &self,
+        queries: Vec<&PreparedStatementWrapper>,
+        params: Vec<Vec<Option<&MaybeUnsetQueryParameterWrapper>>>,
+        options: &QueryOptionsWrapper,
+    ) -> napi::Result<Vec<QueryResultWrapper>> {
+        let params_vec: Vec<Vec<Option<MaybeUnset<CqlValue>>>> = params
+            .into_iter()
+            .map(MaybeUnsetQueryParameterWrapper::extract_parameters)
+            .collect();
+        let mut set: JoinSet<(
+            Result<scylla::response::query_result::QueryResult, scylla::errors::ExecutionError>,
+            usize,
+        )> = JoinSet::new();
+        let mut i = 0;
+        let mut in_flight = 0;
+
+        let mut res: Vec<Option<QueryResultWrapper>> = Vec::new();
+        while res.len() < queries.len() {
+            res.push(None);
+        }
```

Comment:
```suggestion
        res.resize(queries.len(), None);
```

Reply:
Ugh, really? If this is the reason you did it this way, then please write that in a comment.